test/run_test.sh

```diff
@@ -203,6 +208,9 @@ function link_latest() {
 # If an outcome file was produced, keep a link to the latest file
 link_latest outcome json
 
+# If a traffic model file was produced, keep a link to the latest file
+link_latest traffic.model json
```

Comment:
Do you want to diff the latest traffic model file with the previous traffic model file?
We do it with all the other data files, so that any large changes show up in the test output.